### DISCUSSION TITLE
BNT 1

### DISCUSSION THREAD

**Discussion by dicheto** [iptv-org/community#483] [Show and tell]
### Channel ID

BNT1.bg

### Stream URL (optional)

https://de1.glebul.com/hls/hd-bnt-1-hd/index.m3u8?e=1692277851&hash=EYwdqk1gsSj9zO9jXMcssQ

### Notes (optional)

Is this link okay?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://de1.glebul.com/hls/hd-bnt-1-hd/index.m3u8?e=1692277851&hash=EYwdqk1gsSj9zO9jXMcssQ (offline)
